tests/ignite/handlers/test_checkpoint.py

```diff
@@ -1574,3 +1575,21 @@ def test_get_default_score_fn():
     score_fn = Checkpoint.get_default_score_fn("loss", -1)
     score = score_fn(engine)
     assert score == -0.123
+
+
+def test_save_xla_for_disk_saver(dirname):
+    with mock.patch("ignite.handlers.checkpoint.idist.has_xla_support"):
```

Comment:
Instead of mocking we can execute this test with XLA tests as here for example : 
https://github.com/pytorch/ignite/blob/d8e6bec44e8307f22a0c926ef78a07c47255fd19/tests/ignite/handlers/test_checkpoint.py#L1215-L1221

Reply:
@vfdev-5  wouldn't `@pytest.mark.skipif(not idist.has_xla_support, reason="Not on TPU device") ` would be executed only for specific test runs i.e. when we are running tests on TPU device.
My assumption was to actually execute that test regardless of the running device of the test case, in that case the coverage would increase for the `checkpoint` module. Does this make sense?

Reply:
We have XLA tests executed here with GIthub Actions and codecov uploaded, please see : https://github.com/pytorch/ignite/blob/master/.github/workflows/tpu-tests.yml

I think this test have to be executed within the appropriate test suite.